diagrams/addresses.svg

```diff
@@ -0,0 +1,3 @@
IECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIEChHQOCsHHezEiBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAoHICAmeVa5mCCRAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgUI6AwFk57mYlQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIBA5QQEzirXMgUTIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECgHAGBs3LczUqAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAIHKCQicVa5lCiZAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgEA5AgJn5biblQABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABApUTEDirXMsUTIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAgXIEBM7KcTcrAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIEKicgcFa5limYAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC5QgInJXjblYCBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAhUTuD/A3ccQ8ibeRgBAAAAAElFTkSuQmCC" preserveAspectRatio="none"/><path d="M 547.27 218 L 547.27 271.9" fill="none" stroke="#ff0000" stroke-width="3" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 547.27 278.65 L 542.77 269.65 L 547.27 271.9 L 551.77 269.65 Z" fill="#ff0000" stroke="#ff0000" stroke-width="3" stroke-miterlimit="10" pointer-events="all"/><rect x="285.54" y="158" width="523.46" height="60" fill="none" stroke="#ff0000" stroke-width="3" pointer-events="all"/><rect x="284.27" y="282" width="526" height="80" fill="none" stroke="#ff0000" stroke-width="3" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 524px; height: 1px; padding-top: 322px; margin-left: 285px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font face="Verdana">This is where you can register your Ethereum address for your project. FuelStop would only "fund" transactions that are sent from registered addresses. Otherwise, it will simply broadcast the signed transaction to the Ethereum network and behave no differently than regular Ethereum nodes. Before sending your transactions to FuelStop, make sure you have registered the address if the address needs funding.</font></div></div></div></foreignObject><text x="547" y="326" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">This is where you can register your Ethereum address for your project. FuelStop would o...</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://desk.draw.io/support/solutions/articles/16000042487" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
This is where you register your Ethereum addresses for your project. FuelStop will only "fund" transactions that are sent from registered addresses. Otherwise, it will simply broadcast the signed transaction to the Ethereum network and behave no differently than regular Ethereum nodes. Before sending your transactions to FuelStop, make sure you have registered the address if the address needs funding.